src/Samples.Tests/Tests/CreateAndUpdateSubAccountExampleTest.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using OneSpanSign.Sdk;
+
+namespace SDK.Examples
+{
+    [TestFixture()]
+    public class CreateAndUpdateSubAccountExampleTest 
+    {
+    [Test()]
+    public void VerifyResult()
+    {
+        IList<Account> subAccountList;
+        IList<AccessibleAccountResponse> accessibleAccountList;
+        CreateAndUpdateSubAccountExample example = new CreateAndUpdateSubAccountExample();
+
+        try {
+            example.Run();
+        } catch (OssServerException e) {
+            Assert.IsTrue(e.Message.Contains("error.notFound.accountNotFound"));
+            return;
+        } finally {
+            subAccountList = example.subAccounts;
```

Comment:
Same remark as with the java SDK, why use finally if your catch has a return ? Structure feels wrong.

Reply:
Since there isn't any way to get AccountID in SDK (there is Getting Account **private** RestAPI, it isn't exist in SDK), I've to use dummy AccountID in example class, that will cause "accountNotFound" exception. this test should cover this case any way. but if someone uses SDK and update Example to use real AccountID, the Test should work also, that's reason the test use "catch" to cover dummy account test and "finally" to cover real account test.

Reply:
I think it should be documented in the test to avoid confusion on the structure

Reply:
Sure, will be added.